### PR TITLE
pi: add page

### PR DESCRIPTION
**Pull request by neel24**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/pi.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# pi
+
+> Prints 100 decimal digits of Archimedes' constant pi or a number of digits specified by an integer parameter on the command line.
```
Comment (navarroaxel) [inline pages/linux/pi.md:3]: The command could be used for Math operations in Bash, so it's something more than display numbers:
```suggestion
> Compute decimal Archimedes' constant Pi on the command line.
```

File: pages/linux/pi.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <http://manpages.ubuntu.com/manpages/trusty/man1/pi.1.html>.
+
+- Print 100 decimal digits of Archimedes' constant pi:
```
Comment (navarroaxel) [inline pages/linux/pi.md:6]:
```suggestion
- Display 100 decimal digits of Archimedes' constant Pi:
```

File: pages/linux/pi.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`pi`
+
+- Print *number* decimal digits of Archimedes' constant pi:
+
+`pi {{number}}`
+
+- Show summary of options:
```
Comment (navarroaxel) [inline pages/linux/pi.md:14]:
```suggestion
- Display help:
```

File: pages/linux/pi.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`pi --help`
+
+- Show current version of program:
```
Comment (navarroaxel) [inline pages/linux/pi.md:18]:
```suggestion
- Display version:
```

File: pages/linux/pi.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`pi --version`
+
+- Output recommended readings:
```
Comment (navarroaxel) [inline pages/linux/pi.md:22]:
```suggestion
- Display recommended readings:
```

File: pages/linux/pi.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# pi
+
+> Prints 100 decimal digits of Archimedes' constant pi or a number of digits specified by an integer parameter on the command line.
+> More information: <http://manpages.ubuntu.com/manpages/trusty/man1/pi.1.html>.
+
+- Print 100 decimal digits of Archimedes' constant pi:
+
+`pi`
+
+- Print *number* decimal digits of Archimedes' constant pi:
```
Comment (navarroaxel) [inline pages/linux/pi.md:10]:
```suggestion
- Display `number` decimal digits of Archimedes' constant Pi:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @neel24, welcome and thanks for your submission. I left some comments.

**Comment by neel24** [comment]
@navarroaxel, I made the suggested changes with the [Make suggested changes](https://github.com/tldr-pages/tldr/pull/4653/commits/310b2ddfea6d850d56ec7c8a2cc6dd345ade2dbd) commit.

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/pi.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# pi
+
+> Compute decimal Archimedes' constant Pi on the command line.
+> More information: <http://manpages.ubuntu.com/manpages/trusty/man1/pi.1.html>.
+
+- Display 100 decimal digits of Archimedes' constant Pi:
+
+`pi`
+
+- Display `number` decimal digits of Archimedes' constant Pi:
```
Comment (sbrl) [inline pages/linux/pi.md:10]: Perhaps this will read better?
```suggestion
- Display a specified number of decimal digits of Archimedes' constant Pi:
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new page, @neel24 ! I've left a comment below for you to review.

**Review by sbrl — APPROVED** [review]
Looks ok to me! Thanks, @neel24 😺

**Pull request merged**